lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -627,6 +729,21 @@ struct ClassExternFieldOpConversion
   }
 };
 
+struct ObjectOpConversion : public OpConversionPattern<om::ObjectOp> {
```

Comment:
I needed something like this as well, this is slightly different than what I did, but I think has the same effect--we just need the ObjectOp to use the type converted operands, right?